src/components/FilmCardComponent/FilmCard.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { FilmCard } from "./FilmCard";
+import { shallow } from 'enzyme';
+
+describe('<FilmCard>', () => {
+    let wrapper;
+
+    const props = {
+        poster_path: 'poster',
+        title: 'title text',
+        release_date: '2323-20-20',
+        genres: ['genres', 'genres', 'genres'],
+        genre_ids: [1, 2, 3, 4],
+    };
+
+
+    beforeEach(() => {
+        wrapper = shallow(<FilmCard {...props} />);
+    });
+
+    it('should render <FilmCard/> correctly', () => {
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    it('should render img', () => {
+        expect(wrapper.find('#filmCardImg')).toHaveLength(1);
+    });
+
+    it('should render h5', () => {
+        expect(wrapper.find('#filmCardTitle')).toHaveLength(1);
+    });
+
+    it('should render p (date)', () => {
```

Comment:
Please name test title correctly. For example 'should render film year correctly'